rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -174,21 +180,68 @@ void logUpdateCollections() {
         assertEvent(eventLogs, 0, BridgeEvents.UPDATE_COLLECTIONS.getEvent(), new Object[]{}, new Object[]{tx.getSender(signatureCache).toString()});
     }
 
-    @Test
-    void logAddSignature() {
-        // Setup logAddSignature params
-        BtcECKey federatorPubKey = BtcECKey.fromPrivate(BigInteger.valueOf(2L));
+    private static Stream<Arguments> logAddSignatureArgProvider() {
+        ActivationConfig.ForBlock fingerrootActivations = ActivationConfigsForTest.fingerroot500().forBlock(0);
+        ActivationConfig.ForBlock arrowHeadActivations = ActivationConfigsForTest.arrowhead600().forBlock(0);
+
+        FederationMember multiKeyFedMember = new FederationMember(
+            BtcECKey.fromPrivate(BigInteger.valueOf(100)),
+            ECKey.fromPrivate(BigInteger.valueOf(200)),
+            ECKey.fromPrivate(BigInteger.valueOf(300))
+        );
+        ECKey rskKeyFromMultiFedMemberBtcKey = ECKey.fromPublicOnly(multiKeyFedMember.getBtcPublicKey().getPubKey());
```

Comment:
This is not the rsk key, it's the btc one. And why not simply put it in a variable and use that in FederationMember constructor? Looks weird to first set it and then get it

Reply:
_[marcos-iov] [5 hours ago]
This is not the rsk key, it's the btc one._

but isn't that the way it was being done?

Reply:
Fixed!

Reply:
@julia-zack the problem is with the variable name, it's called rskKey and getting the btc key

Reply:
but that was not the way that the rsk key was being obtained? "converting" the btc key in an ECKey?